src/components/App.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { browserHistory } from 'react-router'
+import { connect } from 'react-redux'
+import LogoutPage from './LogoutPage'
+import PROJECTApp from './PROJECTApp'
+
+const App = ({isLoggedIn}) => (
+  <div>
+    {!isLoggedIn && browserHistory.push('/login') }
+    {isLoggedIn && <LogoutPage />}
+    {isLoggedIn && <PROJECTApp />}
+  </div>
+);
+
+const mapStateToProps = state => {
+  return {
```

Comment:
In case you need to return single line you can directly assign it to the arrow function.
No need to add return to it.